lib/graphql/query/validation_pipeline.rb

```diff
@@ -16,7 +16,6 @@ class Query
     class ValidationPipeline
       def initialize(query:, validate:, parse_error:, operation_name_error:, max_depth:, max_complexity:)
         @validation_errors = []
-        @analysis_errors = []
```

Comment:
This is actually not used anymore, since the analysis error logic is handled in `analyze_query`!